spec/features/users/user_can_create_account_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe "When user clicks create account" do
+  scenario "they are able to create an account" do
+    # visit login_path
```

Comment:
I commented this out because it follows the user story, but can't be implemented until we make the sessions controller.